lib/mocha/class_method.rb

```diff
@@ -108,7 +106,7 @@ def store_original_method_visibility
       @original_visibility = method_visibility
     end
 
-    def stub_method_overwrites_original_method?
+    def stub_method_overwrites_original?
```

Comment:
I'd prefer if the renaming of this method could be moved into a separate commit.